lib/dialogs/promo/connected_wallet_promo_dialog.dart

```diff
@@ -0,0 +1,134 @@
+    var wid = 500.0;
+    const edge = 40.0;
+    if (_width < 505) wid = _width;
+    var hgt = 345.0;
+    if (_height < 350) hgt = _height;
+
+    return Dialog(
+      backgroundColor: Colors.transparent,
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(12),
+      ),
+      child: Container(
+        height: hgt,
+        width: wid,
+        decoration: boxDecoration(Colors.grey[900]!, 30, 0, Colors.black),
+        child: Center(
+          child: SizedBox(
+            height: 275,
+            width: wid - edge,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                SizedBox(
+                  width: wid - edge,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      const SizedBox(width: 5),
+                      const Text(
+                        'Wallet successfully connected!',
+                        style: TextStyle(color: Colors.white, fontSize: 20, fontWeight: FontWeight.bold),
+                      ),
+                      SizedBox(
+                        width: 40,
+                        child: IconButton(
+                          icon: const Icon(
+                            Icons.close,
+                            color: Colors.white,
+                          ),
+                          onPressed: () => Navigator.pop(context),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Container(
+                      width: 55,
+                      height: 55,
+                      decoration: const BoxDecoration(
+                        shape: BoxShape.circle,
+                        image: DecorationImage(
+                          image: AssetImage(
+                            'assets/images/x.jpg',
+                          ),
+                        ),
+                      ),
+                    )
+                  ],
+                ),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    const Text(
+                      'Did you learn about AX from an existing AX User?',
+                      style: TextStyle(
+                        fontSize: 16,
+                        color: Colors.white,
+                      ),
+                      textAlign: TextAlign.center,
+                    ),
+                    const SizedBox(height: 15),
+                    const Text(
+                      ' Use the following link to join our discord'
+                      ' and shout them out and both of you will receive'
+                      ' 1000 AX!',
```

Comment:
nit: we should really consider doing localization for all of our texts in the app so we can easily support other languages https://docs.flutter.dev/development/accessibility-and-localization/internationalization

Reply:
Yeah we can have a ticket that will localize the text for every feature